devel/fortio/http.go

```diff
@@ -710,3 +711,69 @@ func DynamicHTTPServer() int {
 	}(port)
 	return port
 }
+
+// DebugHandler returns debug/useful info
+func DebugHandler(w http.ResponseWriter, r *http.Request) {
+	LogVf("%v %v %v %v", r.Method, r.URL, r.Proto, r.RemoteAddr)
+	var buf bytes.Buffer
+	buf.WriteString("Φορτίο version ")
```

Comment:
cool ! I didn't know about text/template ! let me try this (for debug the perf doesn't matter so it should be fine)